### DISCUSSION TITLE
where is documentation for 0.7.1

### DISCUSSION THREAD

**Discussion by littlegreenbean33** [duckdb/duckdb#6735] [Q&A]
I see the website references only dev or 0.6.1 docs ? is that by design ?

**Comment by Tishj** [comment]
We need to update the version on the docs, but everything you see on `dev` is what's guaranteed to be on master.
So not everything you see on `dev` is guaranteed to be available in 7.1

**Reply by Tishj** [reply]
This is no longer true, we recently properly split the `dev` into 0.7.1 archive and dev :)
